src/main/java/frc/robot/RobotContainer.java

```diff
@@ -58,10 +58,12 @@ public RobotContainer() {
 
     robotContext = new RobotContext(arm);
 
-    autoGenerator = Constants.PathPlanner.PATH_PLANNER_IS_ENABLED
-
-        ? new PathPlannerAutoGenerator(drive, arm, shooter, undertakerSubsystem, robotContext)
-        : new SimpleAutoGenerator(drive, shooter, undertakerSubsystem, robotContext);
+    autoGenerator = 
+      Constants.PathPlanner.PATH_PLANNER_IS_ENABLED
+      ?
```

Comment:
if a ternary is this long i'd prefer you just do it the long way, lol

Reply:
Up to you though!

Reply:
Hold on this isn't even for your changes. Don't do this in this PR please.

Reply:
i accidentally modified it in this PR without meaning to...(auto format) which is why I set it back

Reply:
ohhh sorry